slices/libpq5.yaml

```diff
@@ -0,0 +1,17 @@
+package: libpq5
+
+essential:
+  - libpq5_copyright
+
+slices:
+  libs:
+    essential:
+      - libc6_libs
+      - libgssapi-krb5-2_libs
+      - libldap2_libs
+      - libssl3t64_libs
+    contents:
+      /usr/lib/*-linux-*/libpq.so.5*:
+  copyright:
+    contents:
+      /usr/share/doc/libpq5/copyright:
```

Comment:
Do we really need the copyright?

Reply:
I was following the changes from 5797b6dc28eb23f0150b29450983ba1b769b60b9

Reply:
indeed. ty @endersonmaia